### DISCUSSION TITLE
UI freezes when a paper with 200+ cells is cleared

### DISCUSSION THREAD

**Discussion by kvinita** [clientIO/joint#2573] [Q&A]
### Introduction

I have a paper with over 200+ cells on it. When I try to clear the paper programmatically using     `this.rappid.graph.clear();`, then the UI freezes. 

Is there a way to overcome this?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Cannot reproduce (removing cells from the graph / paper is really fast).

You can try to use `resetCells()` instead. The methods triggers only `reset` event on the graph, while the `clear()` method triggers `remove` event on every single cell.
```js
graph.resetCells([])l
```
It's possible that you perform a heavy operation on `remove` event.
